src/components/text-editor/prosemirror-adapter/menu/types.ts

```diff
@@ -15,6 +15,10 @@ export const EditorMenuTypes = {
     Link: 'link',
     OrderedList: 'ordered_list',
     BulletList: 'bullet_list',
+    Underline: 'underline',
+    Strikethrough: 'strikethrough',
```

Comment:
I'm getting errors like these when using underline and strikethrough:

<img width="551" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/8ecf5b14-b5b3-419d-bed0-eb3c26611f3f">

Underline and strikethrough looks like they are working in the editor, but as soon as you have any part of the text formatted with either of them, the output stops updating.

I don't think we should spend time debugging this until we know whether underline and strikethrough are in the set of MVP features though.

Reply:
It's a very small fix for these two. The marks for underline and strikethrough aren't in the default markdown serialiser that prosemirror provides.

Reply:
OK, if you know what's needed, and it doesn't turn into a rabbit hole (aka, you can fix it before the meeting at 16:00), go ahead 😊

Reply:
(To clarify, I don't mean to take the role of some ultimate dictator, deciding what you guys can and cannot do. My intention is just to provide a firm guiding hand 😊)